estargz/types.go

```diff
@@ -159,7 +159,7 @@ type TOCEntry struct {
 
 	// NumLink is the number of entry names pointing to this entry.
 	// Zero means one name references this entry.
-	NumLink int
+	NumLink int `json:"-"`
```

Comment:
Maybe add a comment line to explain the reason for this

Reply:
Thanks for the review. Added the comment.